### ISSUE TITLE
Pusher Robot Not Properly Resetting

### ISSUE THREAD

**Issue by EthanWJohnson** [phetsims/forces-and-motion-basics#203]
Tested on Chrome, Firefox, Edge, Internet Explorer

OS: Reproduced on Windows 10 and Windows 7

Description: After moving the payload/objects on the friction and acceleration screens, resetting, then attempting to push a heavy load while not overcoming friction will cause the pusher to move to pre-reset position, relative to the payload/objects. 

Gif explaining what's going on;
![pusher problems](https://cloud.githubusercontent.com/assets/22551096/19251344/069b79a8-8efc-11e6-8eae-ee338f02c2b9.gif)

Steps to Reproduce:

1.)Enter the "Friction" or "Acceleration" screen

2.)Push any object any distance with the pusher

3.)Reset the screen 

4.)Drag any object to the screen/set the friction slider so that the pusher cannot overcome friction with any force.

5.)Apply a force to the objects in either direction, then release it

Upon releasing the force, the pusher will snap to its position prior to hitting the reset button

Additional Notes:
-Pusher may be located to a position on screen or off-screen, potentially very far off-screen depending on prior position
-Pusher properly moves/resets should the objects overcome friction and actually move

Reference: https://github.com/phetsims/tasks/issues/697

Troubleshooting information (do not edit):
Name: ‪Forces and Motion: Basics‬
URL: http://www.colorado.edu/physics/phet/dev/html/forces-and-motion-basics/2.1.0-rc.3/forces-and-motion-basics_en.html
Version: 2.1.0-rc.3 2016-10-06 22:01:12 UTC
Features missing: touch
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/53.0.2785.143 Safari/537.36
Language: en-US
Window: 1366x648
Pixel Ratio: 1/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 16384x16384
OES_texture_float: true
Dependencies JSON: {"assert":{"sha":"7d27130a","branch":"HEAD"},"axon":{"sha":"e0192608","branch":"HEAD"},"babel":{"sha":"740673a2","branch":"master"},"brand":{"sha":"f0b1f7da","branch":"HEAD"},"chipper":{"sha":"07058555","branch":"HEAD"},"dot":{"sha":"39436598","branch":"HEAD"},"forces-and-motion-basics":{"sha":"4d36bb54","branch":"HEAD"},"joist":{"sha":"b513b5ee","branch":"HEAD"},"kite":{"sha":"73302899","branch":"HEAD"},"phet-core":{"sha":"c48bf320","branch":"HEAD"},"phetcommon":{"sha":"83ea84c8","branch":"HEAD"},"query-string-machine":{"sha":"05231e54","branch":"HEAD"},"scenery":{"sha":"e78ee413","branch":"HEAD"},"scenery-phet":{"sha":"f0fc9ae8","branch":"HEAD"},"sherpa":{"sha":"bcc28cd6","branch":"HEAD"},"sun":{"sha":"f2bd9d60","branch":"HEAD"},"tandem":{"sha":"d369b847","branch":"HEAD"},"vibe":{"sha":"b422db9c","branch":"HEAD"}}

**Comment by jessegreenberg** [comment]
It looks like I forgot to reset a variable used in the translation calculation

```js
      var modelDelta = -( model.position - model.previousModelPosition );
```

model.previousModelPosition is not reset when 'Reset All' button is pressed, so the pusher is getting translated by this when it can't push the stack after reset is pressed.

**Comment by jessegreenberg** [comment]
Thanks for catching this @EthanWJohnson. This should be fixed in the commit above, leaving open to verify.

**Comment by EthanWJohnson** [comment]
Tested on multiple devices, verified fixed now.

**Comment by jessegreenberg** [comment]
Thanks for checking @EthanWJohnson!  I am going to reopen this issue for now to verify that it has also been fixed correctly in the 2.1 branch in the next RC.

**Comment by phet-steele** [comment]
Looks bueno in 2.1.0-rc.5.
